frontend/src/ui/section/profile/Profile.scss

```diff
@@ -123,29 +123,25 @@
   }
 
   .languages {
-    &__title-container {
-      display: flex;
-      color: $rindus-dark-gray;
-      padding: 12px 16px;
-      align-items: center;
-      gap: 16px;
-      border-bottom: 1px solid $rindus-light-gray;
-    }
+    padding: 8px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
     &__title {
-      color: $rindus-dark-gray;
-      font-family: $font-poppins-regular;
-      font-size: 16px;
-      font-style: normal;
-      font-weight: 400;
-      line-height: 22px;
+      padding-left: 16px;
+    }
+    &__title-container {
+      color: #303140;
```

Comment:
1- The color shouldnt be in the title-container should be in the container.

2-The color should use the variables.

3-All these properties are deleted after the refactor and we should keep it.

font-family: $font-poppins-regular;
      font-size: 16px;
      font-style: normal;
      font-weight: 400;
      line-height: 22px;